test/test_sparse.py

```diff
@@ -4049,6 +4049,58 @@ def test_basic(self):
         self.assertEqual(r.values(), torch.empty(0, 4, device='meta'))
 
 
+class TestSparseAny(TestCase):
+
+    def test_generate_simple_inputs(self):
+        # Temporarily disable BSC and BSC layouts as these don't support select yet
+        layouts = [torch.strided, torch.sparse_coo, torch.sparse_csr, torch.sparse_csc, torch.sparse_bsr, torch.sparse_bsc][:-2]
```

Comment:
Is the select requirement coming from `assertEqual`? 

Is there some workaround we could use to test the generator for those layouts? I don't feel great about them being available to pull from the generator without testing.